scripts/zones/Abyssea-Grauberg/MobIDs.lua

```diff
@@ -0,0 +1,29 @@
+require("scripts/globals/keyitems");
+
+-- [QM] = {'name', {list of required trades}, {list of required KIs}, nmID}
+-- uncomment lines as NMs are implemented
+NM_POPS = 
+{
+--  [17818081] = { 'qm1',           {3260},                                                {}, 17818041}, -- Ironclad Sunderer
```

Comment:
http://ffxiclopedia.wikia.com/wiki/Ironclad_Sunderer
Should require Teekesselchen Fragment and Darkflame Arm in trade